### ISSUE TITLE
Ability to upload a user photo

### ISSUE THREAD

**Issue by thatasko** [pagekit/pagekit#484]
Using Gravatar these days is a bit harsh on the user as it requires a third party account (wordpress.com) and not eveybody is thrilled about it. That's probably why Github itself as of relatively recently allows the user to upload a photo as well, and I think this exact same feature should also be present in Pagekit. What do you guys think?

**Comment by saschadube** [comment]
Yes, that is a good idea. I can put it on the todo list or if someone want to help out with this, any pull request is much appreciated.

**Comment by michael-tourneur** [comment]
@saschadube i will work if needed

**Comment by stefanpkc** [comment]
where should the image be saved? storage? maybe also some permissions on who can access this image in the admin area if storage is being used? just raising some questions ;-)

**Comment by PremadeGames** [comment]
I second this! I don't like to use Gravatar these days.. I don't think it should be neccesary to register on a different website so you have an avatar on your own.

**Comment by noBlubb** [comment]
@stefanpkc Anyone with your email may lookup gravatar and nobody is required to upload a custom avatar. I would vote for the "simple" route and put it into storage (without further protection).
